source/branding.html.erb

```diff
@@ -55,8 +55,16 @@ title: Brand Guidelines
     </p>
   </div>
   <div class="small-12 large-6 columns">
-    <%= chef_logo(:span, :style => "margin:22px;") %>
-    <%= image_tag "example/logo-with-io.png", :width => 85, :style => "vertical-align:top;margin:22px;" %>
+    <div class="row" style="text-align:center;">
+      <div class="small-6 large-6 columns">
```

Comment:
I don't believe you need to specify a large class if it's the same column count as the small.